libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts

```diff
@@ -93,30 +107,27 @@ function addStylesheetToWorkspace(): Rule {
     });
 }
 
-export default function (): Rule {
-  return (tree) => {
-    const packageJson: {
-      dependencies?: { [_: string]: string };
-      devDependencies?: { [_: string]: string };
-    } = JSON.parse(readRequiredFile(tree, '/package.json'));
-
-    let contents = '';
-
-    for (const library of compatStyles.libraries) {
-      if (
-        packageJson.dependencies?.[library.name] ||
-        packageJson.devDependencies?.[library.name]
-      ) {
-        for (const component of library.components) {
-          contents += buildComponentCss(component);
-        }
+function getCompatStyles(tree: Tree): string | undefined {
+  let contents = '';
+
+  const packageJson: {
+    dependencies?: { [_: string]: string };
```

Comment:
While you're in there, could you change these to `Record<string, string>`?

Reply:
Good call. Done.